priv/posts/eef/20210902233320_agm-2021.md

```diff
@@ -0,0 +1,67 @@
+{
+  "title": "EEF 2nd. Annual General Meeting",
+  "authors": ["eef"],
+  "slug": "agm-2021",
+  "category": "eef",
+  "tags": ["eef", "agm"],
+  "datetime": "2021-09-02T23:33:20.702576Z"
+}
+---
+ EEF 2nd. Annual General Meeting
+---
+
+## Zoom Conference Call Times
+
+Thursday Sept 9th, 20:00 CET / 18:00 UTC / 09:00 USA Pacific Time
+
+**Meeting A:** EEF Annual General Meeting 2021 EMEA / Americas
+Time: Thursday, Sep 9th, 2021 20:00 CET / 18:00 UTC / 09:00 USA Pacific Time 
+
+Members will receive an email with a link and credentials for the meetings. 
+
+**Meeting B:** EEF Annual General Meeting 2021 EMEA / APAC
+Friday: Sep 10th, 2021 15:00 Japanese Time; 08:00 CET; 06:00 UTC
+
+Members will receive an email with a link and credentials for the meetings. 
+
+## Quorum Requirements
+Our Bylaws Section 3.9 require a quorum of 1/3 (one third) of the eligible voting members to be present for a meeting to be deemed quorate.
+Since:
+1. Covid-19 restrictions prohibit an in-person AGM, and
+2. Global distribution of the members make real-time synchronization due to time-zone awkward,
+
+the board decided per Section 3.1 to hold the AGM electronically. In order to comply with the three subsections of Section 3.1 that require authentication of the voting members and allowing members reasonable access to the content, we will be holding the actual voting via the Wild Apricot Members portal (authentication) and two Zoom Conferences (content access) as a mechanism of interactive feedback and Q&A.
+
+**Note:** since the AGM is electronic and virtual, Section 14.8 Electronic Voting rules apply.
+Proposed AGM Process
+1. Electronic voting will open on September 11th.
```

Comment:
which timezone? We usually work on Pacific Time since that's where we're incorporated. Check out the election posts for the choice there.

Reply:
Hmm, this should be the 9th I think actually. We didn't specify a timezone in last year's post, but I agree pacific time makes sense to me.